pages/gm/f22.js

```diff
@@ -0,0 +1,208 @@
+      </div>
+    </div>
+    <div className="text-center">
+      <h1>Fall 2022 General Meeting</h1>
+      <Link href={RSVP_LINK}>
+        <a className="button" target="_blank" rel="noreferrer noopener">
+          RSVP Now!
+        </a>
+      </Link>
+    </div>
+    </>
+  );
+}
+
+function F22() {
+	return (
+		<Layout>
+			<NextSeo
+				title="Fall General Meeting | ACM at UCLA"
+				description="ACM's Fall General Meeting happened on September 26th at 6 PM PT!"
+				openGraph={{
+					images: [
+						{
+							url: 'https://www.uclaacm.com/images/logo.png',
+							width: 1200,
+							height: 1200,
+							alt: 'The ACM at UCLA logo',
+						},
+					],
+					site_name: 'ACM at UCLA',
+				}}
+			/>
+			<Banner decorative />
+      <div className="text-center">
+        <Image
+          src = {fallGMgraphic}
+          className="gm-graphic"
+          alt="Fall GM 2021 Marketing Graphic. Fall GM will happen on Monday, september 26th from 6pm to 8pm in Ackerman Grand ballroom"/>
+      </div>
+      <Countdown date={GM_START_TIME} renderer={countdownRenderer}/>
+			<div className="content-container-tight text-center">
+        <div id="info-wrapper">
+          <h2>Relevant information</h2>
+          <div className="flex">
+            <div>
+              <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d13219.85897956824!2d-118.4441451!3d34.070418!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x62f97fe423993f80!2sUCLA%20Ackerman%20Union!5e0!3m2!1sen!2sin!4v1663063464157!5m2!1sen!2sin" width="450" height="400" style={{border: 0}} allowFullScreen="" loading="lazy" title="gogole maps embed of court of sciences"></iframe>
+            </div>
+            <div className="what-to-bring">
+              <h3>How to get there</h3>
+              <p> Fall GM will be hosted in the Grand Ballroom on the second floor of the Ackerman Union building. </p>
+              <br></br>
+              <h3>What to bring</h3>
+              <p>Recommended: Face Mask.</p>
+              <p>Encouraged: Phone to scan QR codes, excitement to learn about ACM!</p>
+            </div>
+          </div>
+          <p> Don&apos;t hesitate to contact us at acm@ucla.edu if you any accessiblity concerns for fall GM.</p>
+
+        </div>
+
+        <div id="gm-program-wrapper">
+          <h2>Program</h2>
+					<div className="gm-program-section">
+            <div className="section-header">
+              <h3>Welcome</h3>
+              <p className="sub-heading">An introduction to ACM by our president Ava Asmani.</p>
+            </div>
+					</div>
+          <div className="gm-program-section">
+            <div className="section-header">
+              <h3>Committee Presentations</h3>
+              <p className="sub-heading">Learn what ACM&apos;s eight committees have planned for Fall quarter.</p>
+            </div>
+            <p className="gm-program-row"><Image src= {studioLogo} alt="ACM studio"/> Caroline Wang, Ray Hsiao</p>
```

Comment:
For all the images, it might be a good idea to set the layout property of the image, as specified in the documentation: https://nextjs.org/docs/api-reference/next/image

setting the layout property and specifying the dimensions of the image with width and height could help with our site's responsiveness.